base/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+from .models import task
+admin.site.register(task)
```

Comment:
please follow PEP8 conventions here. there should be  2 blank line between import block and the rest of the code